gazelle/python/testdata/remove_invalid_library/BUILD.in

```diff
@@ -0,0 +1,7 @@
+load("@rules_python//python:defs.bzl", "py_library")
+
+py_library(
+    name = "remove_invalid_library",
+    srcs = ["__init__.py"],
+    visibility = ["//:__subpackages__"],
+)
```

Comment:
I am wondering that maybe we should add an extra test that has no `srcs` entries, but only `deps`. This is sometimes used to bundle multiple libraries together and I am wondering if by us changing the behaviour here makes it impossible to have such `py_library` targets.

Reply:
Has been added. But in fact, this `deps_with_no_srcs_library` will not be modified regardless of any content, because this name is user-defined, and gazelle will not generate a rule with the same name, so it will not trigger the action of merge.

Reply:
It has been added. But in fact, this deps_with_no_srcs_library will not be modified regardless of any content, because this name is user-defined, and gazelle will not generate a rule with the same name, so it will not trigger the action of merge.